blaze/expr/expressions.py

```diff
@@ -845,6 +945,7 @@ def ndim(expr):
 schema_method_list.extend([
     (isscalar, set([label, relabel, coerce])),
     (isrecord, set([relabel])),
+    (lambda ds: isinstance(ds, Option), {coalesce}),
```

Comment:
it is not meaningful to coalesce a non-optional type with anything because it is just the identity.